src/layouts/EditContactUs.jsx

```diff
@@ -180,10 +182,11 @@ export default class EditContactUs extends Component {
         errors: {
           contacts: contactsErrors,
           locations: locationsErrors,
-        }
+        },
       });
     } catch (err) {
       console.log(err);
+      this.setState({ shouldRedirectToNotFound: true })
```

Comment:
Currently, error-handling for this layout is lacking in sophistication. We'll handle this in a separate PR, documented in issue #284